global_config.py

```diff
@@ -0,0 +1,29 @@
+from passlib.context import CryptContext
+
+user_conf_path = "./canvas/user_configs/"
+user_cache_path = "./canvas/user_caches/"
+
+front_end_domain = "https://canvashelper.netlify.app"  # Used to set CORS and redirect to root path

```

Comment:
What is this url?

Reply:
This was used for setting CORS for all cookies. And CORS shouldn't be changed to wildcard mark "*" because CORS protected against CSRF attacks. This URL was also used for redirecting user. But I **just** changed the tokens to be stored not in cookie but instead in localStorage, and include it in Authorization header, as OAuth2 recommend, so this url will be removed.